help/articles/101013/index.html

```diff
@@ -37,8 +37,8 @@ <h3>Why reporting content is important</h3>
         <p>We try our very best to try and keep Hydraulisc clean and safe for everyone. That said, we cannot accomplish that goal alone.</p>
         <p>That's where you, our loyal users, come in. By reporting content, we can detect and remove potentially inappropriate content from the platform faster.</p>
         <div class="info">
-            <p>We have a Zero Tolerance policy for any kinds of 18+/sexually explicit (adult) contents.</p>
-            <p>We do not allow for inappropriate Profile Pictures, Posts (Images or Text Descriptions), Usernames, Banners, User Biographies, external links or sexually explicit Private Messages, including 18+ Adult Contents, Sexually Explicit Contents (including Images in Posts, Private Messages and Banners, Text Descriptions of Posts or Private Messages) or Illegal Substances (images, references and discussions).
+            <p>We have a Zero Tolerance policy for any kinds of 18+/sexually explicit (adult) content.</p>
+            <p>We do not allow inappropriate Profile Pictures, Posts (Images or Text Descriptions), Usernames, Banners, User Biographies, external links or sexually explicit Private Messages, including 18+ Adult Contents, Sexually Explicit Contents (including Images in Posts, Private Messages and Banners, Text Descriptions of Posts or Private Messages) or Illegal Substances (images, references and discussions). In addition, do not infringe on others copyright.
```

Comment:
I believe copyright and DMCA issues should be in another article, and therefore, I would like to request that you revert the changes related to it.